Whiskey/Functions/Resolve-WhiskeyNuGetPackageVersion.ps1

```diff
@@ -27,8 +27,8 @@ function Resolve-WhiskeyNuGetPackageVersion
         }
         $Version = $NuGetPackage |
             Where-Object { $_ -match $NuGetPackageName } |
-            Select-Object -First 1 |
             Where-Object { $_ -match ' (\d+\.\d+\.\d+.*)' } |
+            Select-Object -First 1 |

```

Comment:
Last-last. After `ForEach-Object { $Matches[1] }`.

Reply:
Oops, fixed. Moved `Select-Object` to the very last pipe.